services/order.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, User, MovieSession
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
+) -> Order:
+    with transaction.atomic():
+        order = Order.objects.create(user=User.objects.get(username=username))
+        if date:
+            order.created_at = date
+            order.save()
+
+        new_tickets = [
+            Ticket(
+                movie_session=MovieSession.objects.get(
+                    pk=ticket["movie_session"]
+                ),
```

Comment:
it's better to set movie_session_id cuz you already have id + avoid unneccessary sql query